implementation/src/main/java/io/smallrye/mutiny/groups/MultiCreate.java

```diff
@@ -462,4 +464,23 @@ public MultiTimePeriod ticks() {
         return Multi.createFrom().iterable(() -> IntStream.range(startInclusive, endExclusive).iterator());
     }
 
+    /**
+     * Creates a {@link Multi} from a <em>resource</em>, generated by a supplier called each individual {@link Subscriber},
```

Comment:
think it needs a "by" or "with" between "supplier" and "called"

Reply:
I've rewritten the whole block.